src/Components/CollectionPage/BooksInCollectionI.js

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
Why is this file named BooksInCollectionI instead of BooksInCollection, was that a typo?